slowloris.py

```diff
@@ -157,18 +117,37 @@ def init_socket(ip: str):
         ctx.verify_mode = ssl.CERT_NONE
         s = ctx.wrap_socket(s, server_hostname=args.host)
 
-    s.connect((ip, args.port))
+    try:
+        s.connect((ip, port))
+    except socket.error as e:
+        logging.error("Failed to connect to %s:%s - %s", ip, port, e)
+        sys.exit(1)
```

Comment:
Awful error handling. Why do you kill the entire script if a single connection fails?